tox.ini

```
@@ -4,6 +4,15 @@ requires =
     tox-uv >=0.2.0
 envlist = py3,pypy3,json_infra,static,tests_pytest_py3,tests_pytest_pypy3
 
+
+[eest]
+runner = uv-venv-lock-runner
+package = editable
+wheel_build_env = .pkg
+python_source_dirs = src/ethereum_spec_tests tests # .github/scripts
```

```suggestion
python_source_dirs = packages/tests tests # .github/scripts
```

or
```python
python_source_dirs = packages/testing tests # .github/scripts
```
if we decide to change it.

---

It changed!